templates/platform_landing_zone/modules/management_groups/main.tf

```diff
@@ -1,6 +1,6 @@
 module "management_groups" {
-  source                              = "Azure/avm-ptn-alz/azurerm"
-  version                             = "0.10.0"
+  source = "git::https://github.com/Azure/terraform-azurerm-avm-ptn-alz.git?ref=feat/assignPermissions" #"Azure/avm-ptn-alz/azurerm"
```

Comment:
We shouldn't have this ref in production. Once the PR is merged for this module we can update to the registry version